ui/src/adapters/api/credentials.ts

```diff
@@ -450,7 +484,7 @@ export async function createAuthQRCode({
   }
 }
 
-const resultCreatedLinkParser = getStrictParser<ResultCreated<ID>>()(
+const resultCreatedParser = getStrictParser<ResultCreated<ID>>()(
```

Comment:
Why change this name? Is it to be reused? Currently it doesn't make grammatical sense.

Reply:
It should be `resultCreatedIDParser`

i.e, what we parse (`ResultCreated<ID>`) plus the suffix `parser`.